lsp/client/package.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "lsp-sample-client",
+    "description": "VSCode part of a language server",
+    "author": "Microsoft Corporation",
```

Comment:
remove author? root project already have it

Reply:
done